### PR TITLE
Implement grip threshold profile setting

### PR DESCRIPTION
**Pull request by FejZa**
## Overview

https://github.com/XRTK/XRTK-Core/pull/767

### CODE REVIEW

File: XRTK.Lumin/Packages/com.xrtk.lumin/package.json [hunk 1]
```diff
@@ -13,13 +13,13 @@
     "reality",
     "lumin"
   ],
-  "version": "0.2.1",
+  "version": "0.2.3",
```
Comment (StephenHodgson) [inline XRTK.Lumin/Packages/com.xrtk.lumin/package.json:16]: Version bump too far. Should bc 0.2.2

File: XRTK.Lumin/Packages/com.xrtk.lumin/package.json [hunk 2]
```diff
@@ -13,13 +13,13 @@
...
   "unity": "2019.4",
   "license": "MIT",
   "src": "Assets/XRTK.Lumin",
   "author": "XRTK Team (https://github.com/XRTK)",
   "dependencies": {
-    "com.xrtk.core": "0.2.1"
+    "com.xrtk.core": "0.2.3"
```
Comment (StephenHodgson) [inline XRTK.Lumin/Packages/com.xrtk.lumin/package.json:22]: Let's revert the package dependency change for now as there aren't any breaking changes. That way we can get the builds to pass and merge this into development first and do our usual workflow

**Review by StephenHodgson — CHANGES_REQUESTED** [review]
See comments

**Comment by FejZa** [comment]
I am not seeing any comments on this PR? What's wrong with it?

**Review by StephenHodgson — APPROVED** [review]

**Pull request merged**